Products/ZenModel/actions.py

```diff
@@ -394,6 +396,37 @@ def _encodeBody(self, body):
             plain_body = MIMEText(body.decode('ascii', 'ignore'))
         return plain_body
 
+    def _evtTimeToUserTz(self, body, zone_name):
```

Comment:
This function locates a single timestamp within the content.  How can we be sure there won't be additional timestamps?